Docs: Add license, code of conduct, Contributing and security guidelines (#163)
AMDEPYC/sev-certify: CODE_OF_CONDUCT.md

## Code of Conduct

### Contributor Code of Conduct

> As contributors and maintainers of the projects under the [sev-certify](https://github.com/AMDEPYC/sev-certify.git) repository, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute. This may be through adding host/guest images of new Linux distribution, adding new test modules, reporting issues, posting feature requests, updating documentation or submitting pull requests.  This project include the use of mkosi tool as found [here](https://github.com/systemd/mkosi).

Author: Ok, I addressed this comment.

## Code of Conduct

### Contributor Code of Conduct

> As contributors and maintainers of the projects under the [sev-certify](https://github.com/AMDEPYC/sev-certify.git) repository, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through adding host/images of new linux distribution, adding new host/guest modules,reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities to any of the projects under the containers umbrella.  These projects include the use of mkosi tool as found [here](https://github.com/systemd/mkosi).

Reviewer: ```suggestion
As contributors and maintainers of the projects under the [sev-certify](https://github.com/AMDEPYC/sev-certify.git) repository, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through adding host/images of new Linux distributions, adding new test modules, reporting issues, posting feature requests, updating documentation, submitting pull requests , and other activities to any of the projects under the containers umbrella. These projects include the use of mkosi tool as found [here](https://github.com/systemd/mkosi).
```

> We are committed to making participation in these projects a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression,sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.

Reviewer: ```suggestion
We are committed to making participation in these projects a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
```

Examples of unacceptable behavior by participants include:

* The use of sexualized language or imagery.
* Personal attacks.
* Trolling or insulting/derogatory comments.
* Public or private harassment.
* Publishing other's private information, such as physical or electronic addresses,
 without explicit permission.
* Other unethical or unprofessional conduct.

> Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct.  By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing their project.  Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.

Reviewer: ```suggestion
Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing their project. Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.
```

This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.

> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting us at snpcert_email(mailto:snpcert_email)

Reviewer 2: My inclination would be to just reference the list of maintainers. That said, ACE says, 
`Yes, you can request a generic email address or make changes related to mail services, including distribution lists, group requests, and service accounts, by submitting a request through the`
[Generic Request - Mail Services form](https://amd.service-now.com/sp?id=sc_cat_item&sys_id=50abe3dd48a9a4002bb8b8c076adec13)
